config.sample.json

```diff
@@ -2,7 +2,6 @@
     "clientId": "00000000000000",
     "clientSecret": "AAAAAAAAAAAAAAAAAAA",
     "redirectURL": "https://YOURNAME.cloudflareaccess.com/cdn-cgi/access/callback",
-    "serversToCheckRolesFor": [
-
-    ]
+    "serversToCheckRolesFor": [],
```

Comment:
This was intentionally left open so that those less experienced with JSON would understand they need to put things in here if they want servers to have their roles checked for